pkgs/by-name/hu/hurl/package.nix

```diff
@@ -9,34 +9,29 @@
   openssl,
   curl,
   versionCheckHook,
+  clang,
+  llvmPackages,
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "hurl";
-  version = "6.1.1";
+  version = "7.0.0";
 
   src = fetchFromGitHub {
     owner = "Orange-OpenSource";
     repo = "hurl";
     tag = version;
-    hash = "sha256-NtvBw8Nb2eZN0rjVL/LPyIdY5hBJGnz/cDun6VvwYZE=";
+    hash = "sha256-dmPXI2RHEi/wcdVVwBRtBgNXyBXFnm44236pqYjxgBs=";
   };
 
-  cargoHash = "sha256-WyNActmsHpr5fgN1a3X9ApEACWFVJMVoi4fBvKhGgZ0=";
-
-  patches = [
-    # Fix build with libxml-2.14, remove after next hurl release
-    # https://github.com/Orange-OpenSource/hurl/pull/3977
-    (fetchpatch2 {
-      name = "fix-libxml_2_14";
-      url = "https://github.com/Orange-OpenSource/hurl/commit/7c7b410c3017aeab0dfc74a6144e4cb8e186a10a.patch?full_index=1";
-      hash = "sha256-XjnCRIMwzfgUMIhm6pQ90pzA+c2U0EuhyvLUZDsI2GI=";
-    })
-  ];
+  cargoHash = "sha256-1bZaSdMJe39cDEOoqW82zS5NvOlZDGe1ia56BjXddyc=";
 
   nativeBuildInputs = [
     pkg-config
     installShellFiles
+    rustPlatform.bindgenHook
```

Comment:
I just tested the build with only `rustPlatform.bindgenHook` added and that seems to be enough - it takes care of making (lib)clang accessible.